types.go

```diff
@@ -141,6 +142,30 @@ func (t *SectorInfo) sealingCtx(ctx context.Context) context.Context {
 	return ctx
 }
 
+func (t *SectorInfo) unsealedRanges(invert bool) []storage.Range {
```

Comment:
this function (and especially its parameter) are kinda confusing. What is being inverted? its not a super intuitive option.

Reply:
Agreed, but it makes sense if you squint at it. I think it's fine with a one-line explainer on top.